### PR TITLE
Fixes #84

### LINKED ISSUE #84: Use environment variables for credentials
https://github.com/Vaibhavi24/Online-Hotel-Management-System/blob/b0e8dd7d67505601a61a4b064089e6defce84894/data.py#L3

### PR DESCRIPTION
**Pull request by contactvaibhavi**
Moved hardcoded configs to environment variables

Used python library decouple and module config to refer the environment variables. Keeping the env variables locally in a .env file.

### CODE REVIEW

**Comment by contactvaibhavi** [comment]
@as1605

File: app.py [hunk 1]
```diff
@@ -117,16 +122,18 @@ def register():
...
     return render_template('register.html', form=form)
 
 
 @app.route('/login', methods=['GET', 'POST'])
 def login():
+    # If the Form is submitted
     if request.method == 'POST':
         # Get Form Fields
         username = request.form['username']
+        session['user_name'] = username
```
Comment (as1605) [inline app.py]: Keep consistent naming `username` vs `user_name`

File: data.py [hunk 2]
```diff
@@ -1,13 +1,9 @@
 from twilio.rest import Client
+from decouple import config
 
-account_sid = 'AC4813fd548f065ee195b1aa2feb96c58f' # Found on Twilio Console Dashboard
-auth_token = 'cda8141ecc84ab6ae0489aacab4d3586' # Found on Twilio Console Dashboard
-myPhone = '+919013908350' # Phone number you used to verify your Twilio account
-TwilioNumber = '+12692206694' # Phone number given to you by Twilio
-
-client = Client(account_sid, auth_token)
+client = Client(config('ACCOUNT_SID'), config('AUTH_TOKEN'))
```
Comment (as1605) [inline data.py]: Prepend TWILIO in environment variables. `TWILIO_AUTH_TOKEN` vs `AUTH_TOKEN`

File: data.py [hunk 3]
```diff
@@ -1,13 +1,9 @@
 from twilio.rest import Client
+from decouple import config
```
Comment (as1605) [inline data.py:2]: Add `decouple` to `requirements.txt`

**Review by as1605 — CHANGES_REQUESTED** [review]

File: data.py (hunk 3 above)
Comment (contactvaibhavi) [reply data.py:2]: Done

File: data.py (hunk 2 above)
Comment (contactvaibhavi) [reply data.py]: Done

File: app.py (hunk 1 above)
Comment (contactvaibhavi) [reply app.py]: Done

**Comment by contactvaibhavi** [comment]
@as1605 Thanks for review, pls see the latest

**Review by as1605 — APPROVED** [review]
lgtm

**Pull request merged**